src/components/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function Button({ title , disabled , clickBehaviour}) {
+    function functionOne(){ console.log("Dit is functie 1");}
+    function functionTwo(txt){console.log(txt);}
+    return (
```

Comment:
Functies zien er goed uit persoonlijk wist ik niet dat onClick in de opdracht moest staan.
Wel heb je bij de ene een parameter en bij de andere niet terwijl ze hetzelfde doen. Voor het resultaat maakt dit niet veel uit.
Voor overzichtelijkheid lijkt het me persoonlijk beter ze in dezelfde stijl te houden.
Dus of bij allebei een parameter of bij beide geen.